services/src/northbound/src/main/resources/northbound.properties

```diff
@@ -12,3 +12,4 @@ server.contextPath=/api/v1
 kafka.flow.topic=kilda.flow
 kafka.health.topic=kilda.health.check
 kafka.northbound.topic=kilda.northbound
+topology.engine.rest.endpoint=http://127.0.0.1:80
```

Comment:
Can we use the domain name instead of IP? I'm adding 'hostname' to both topo-eng and topo-rest